learn/advanced/datatypes.md

```diff
@@ -70,61 +65,127 @@ You can also create [filters](/learn/advanced/filtering_and_faceted_search.md).
 
 A Boolean value, which is either `true` or `false`, is received and converted to a lowercase human-readable text (i.e. `true` and `false`). Booleans can be searched as they are converted to strings.
 
-## Array
+## Object
```

Comment:
All this part describe the old way of handling objects.
In v0.27.0 we're always doing like you explained in the `nested object` part.

Reply:
Ah, brilliant, thanks for pointing that out. 

Just to confirm, this would mean [this section of the spec](https://github.com/meilisearch/specifications/pull/121/files#diff-9c85f6e9818fa52f93114dd8c378870a6822b5cabc13a14993e5d718357f5106R144-R157) (lines 144-157) is out of date with v0.27 behaviour, right?